cmd/sidecars/network-passt-binding/server/server.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"kubevirt.io/kubevirt/cmd/sidecars/network-passt-binding/callback"
+	"kubevirt.io/kubevirt/cmd/sidecars/network-passt-binding/domain"
+
+	vmschema "kubevirt.io/api/core/v1"
+
+	hooksInfo "kubevirt.io/kubevirt/pkg/hooks/info"
+	hooksV1alpha2 "kubevirt.io/kubevirt/pkg/hooks/v1alpha2"
+
+	"kubevirt.io/client-go/log"
+)
+
+type InfoServer struct {
+	Version string
+}
+
+func (s InfoServer) Info(_ context.Context, _ *hooksInfo.InfoParams) (*hooksInfo.InfoResult, error) {
+	infoResult := &hooksInfo.InfoResult{
+		Name: "network-passt-binding",
+		Versions: []string{
+			s.Version,
+		},
+		HookPoints: []*hooksInfo.HookPoint{
+			{
+				Name:     hooksInfo.OnDefineDomainHookPointName,
+				Priority: 0,
+			},
+		},
+	}
+
+	log.Log.Infof("hook info result: %+v", infoResult)
```

Comment:
I still think this one should be dropped. It does not help in any way to understand if the interesting parts of this sidecar works as expected.

Reply:
Done